runtime/runtime-params-estimator/src/gas_cost.rs

```diff
@@ -32,8 +33,9 @@ pub(crate) struct GasCost {
     ///
     /// Each specific cost can use it's own criteria for uncertainty -- the end
     /// result here is just printing UNCERTAIN next to the corresponding cost in
-    /// the output.
-    uncertain: bool,
+    /// the output. `uncertain_message` can be called to display the reason and
+    /// code location of where the uncertainty has been set.
+    uncertain: Option<(&'static str, &'static Location<'static>)>,
```

Comment:
It's probably worth it to add a named struct here, though no strong opinion.

Reply:
No opinion -- renaming types is easy, so I tend to just rename things as I go, when I realize a better name is possible.